llvm/test/CodeGen/RISCV/cheri/float-mem.ll

```diff
@@ -0,0 +1,112 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; Check that we can load/store single-precision floating point values via a
+; capability pointer without capability mode.
+; RUN: llc -mtriple=riscv32 -target-abi ilp32 -mattr=+xcheri,-cap-mode,-f,-d -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV32IXCHERI-ILP32 %s
+; RUN: llc -mtriple=riscv32 -target-abi ilp32d -mattr=+xcheri,-cap-mode,+f,+d -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV32IXCHERI-ILP32D %s
+; RUN: llc -mtriple=riscv64 -target-abi lp64 -mattr=+xcheri,-cap-mode,-f,-d -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV64IXCHERI-LP64 %s
+; RUN: llc -mtriple=riscv64 -target-abi lp64d -mattr=+xcheri,-cap-mode,+f,+d -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV64IXCHERI-LP64D %s
+
```

Comment:
I personally don't like the gap; it's a bit arbitrary whether you group by hybrid/purecap or XLEN first.

Reply:
Will remove, the gap is just there because that's the order in which I added them.